docs/working_with_containers/container-setup.md

```diff
@@ -11,8 +11,11 @@ the Python side.
 
 ## Setup Requirements
 
-To begin using the container there are two open-source applications that must be installed.
+To begin using the container, you have several options. First, you can use Docker together with DevPod. Alternatively,
+you can use the open-source combination of Podman and DevPod.
 
+* Docker (<https://www.docker.com/>) - a popular container platform that uses a background service (daemon) to build,
+run, and manage containerized applications.
```

Comment:
please explicitly mention that Docker Desktop requires a paid subscription for commercial usage.